src/resources/Customer.js

```diff
@@ -329,6 +350,26 @@ class Customer extends Resource {
     return this.resource(Trip, Trip.makeHref(this.code, id));
   }
 
+  /**
+   * Gets a user resource by id
+   * @param {Object} id Identity of the user or the new object
+   * @returns {User} User resource
+   */
+  user(id) {
```

Comment:
These methods are already on `src/resources/index.js` because the URLs/hrefs do not include customer codes.